third_party/CMakeLists.txt

```
@@ -83,6 +83,11 @@ endif()
 
 if (NOT TARGET glslang)
   if (IS_DIRECTORY ${SHADERC_GLSLANG_DIR})
+    if (SHADERC_ENABLE_TESTS)
+      # Glslang tests are off by default. Turn them on if testing Shaderc.
+      set(GLSLANG_ENABLE_TESTS ON)
```

I think the option is incorrectly named. The glslang CMakeLists.txt has:
>option(GLSLANG_TESTS "Enable glslang testing" ${GLSLANG_TESTS_DEFAULT})

```suggestion
      set(GLSLANG_TESTS ON)
```

---

tested that, and it works. thanks
Tests fail, so we know they are running